Upstream commit replacing a deprecated librsvg function causes svg icons
with a viewbox attibute to be displayed smaller than intended. This patch
will be submitted upsteam.
See: https://lists.sr.ht/~leon_plickat/lavalauncher/%3CCAJC6E0yjCAME1qo1tMxTMuwisqMmMSOOUn7F81x%2B8bNVpwy99g%40mail.gmail.com%3E
--- lavalauncher-v2.1.1/src/types/image_t.c.orig
+++ lavalauncher-v2.1.1/src/types/image_t.c
@@ -212,8 +212,14 @@
 			cairo_scale(cairo, (float)width / rsvg_width.length,
 					(float)width / rsvg_height.length);
 			GError *gerror = NULL;
+			RsvgRectangle viewport = {
+				.x = 0.0,
+				.y = 0.0,
+				.width = rsvg_width.length,
+				.height = rsvg_height.length,
+			};   
 			rsvg_handle_render_document(image->rsvg_handle, cairo,
-					&viewbox, &gerror);
+					&viewport, &gerror);
 			// TODO check value of gerror
 		}
 	}
